activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -46,6 +50,13 @@ def on_load(name, options = {}, &block)
       @load_hooks[name] << [block, options]
     end
 
+    # Executes all blocks which are registered by the on_load method for the
+    # +name+ against the +base+ class.
```

Comment:
What do you think of the following tweak?  (Note that `base` [does not necessarily have to be a class](https://github.com/rails/rails/blob/51b4370bb371a0f1d0823c57cad3a1557bf1e6fe/activesupport/lib/active_support/lazy_load_hooks.rb#L70-L74). :smiley:)

```suggestion
    # Executes all blocks registered to +name+ via on_load, using +base+ as the
    # evaluation context.
```

Reply:
Ah, correct. Force of habit to say "base class" 😃